libgalois/src/PropertyGraph.cpp

```diff
@@ -448,33 +564,25 @@ katana::PropertyGraph::Validate() {
   return katana::ResultSuccess();
 }
 
+/// Converts all uint8/bool properties into EntityTypeIDs
+/// Only call this if every uint8/bool property should be considered a type
 katana::Result<void>
 katana::PropertyGraph::ConstructEntityTypeIDs() {
```

Comment:
Why is this not deleted? Where is this used?

Reply:
This is used when loading an old version of the graph, for backwards compatibility.

Reply:
This is used when loading an old graph, for the sake of backwards compatibility.

Reply:
The enterprise repo is the only user of this now